site/web/assets/js/classification.js

```diff
@@ -1,4 +1,8 @@
-var insertClassification = function (classificationDetails) {
```

Comment:
Shouldn't this file be project_settings.js?

Reply:
This file contains contributions from @sandipbhuyan. I think it won't be wise to rename it or would it be?  After the build, it is transformed into `project_settings` anyways.